tests/utils/log/test_es_task_handler.py

```diff
@@ -81,6 +85,17 @@ def setUp(self):
         self.ti.try_number = 1
         self.ti.state = State.RUNNING
         self.addCleanup(self.dag.clear)
+        self.index_name2 = "test_index2"
+        self.es_task_handler2 = ElasticsearchTaskHandler(
+            self.local_log_location,
```

Comment:
can you rename this to be more specific `es_task_handler2`?

Reply:
can you please suggest anything more meaningful. I would be happy to accommodate. it is literally handler2 with test_index2. Because the index name is part of the handler, we need to use a separate index in order to use a different index name here.